libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/DocumentPermissions.java

```diff
@@ -78,6 +79,8 @@ protected boolean getStandardPermissions(RequestedAction action) {
             case USERS:
             case CLEARING:
                 return PermissionUtils.isAdmin(user) || isModerator();
+            case WRITE_ECC:
+                return PermissionUtils.isAdmin(user);
```

Comment:
That's the thing: I didn't want to allow `WRITE_ECC` for other documents until there's a need to. So, I imagined, if ECC fields are added to some other object in the future, whoever adds the fields, will update the permissions. I didn't want to make them standard just now. But then, I can totally see your point. I think we're splitting hairs here. There isn't a measurable difference right now, just in the potential future. Don't you think?